maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -1181,6 +1181,9 @@ private RunResult executeProvider(
         } else {
             ForkConfiguration forkConfiguration = createForkConfiguration(platform, resolvedJavaModularityResult);
             if (getConsoleLogger().isDebugEnabled()) {
+                getConsoleLogger()
+                        .debug("Using fork starter with configuration "
```

Comment:
The statement is a bit misleading. It does not print the configuration, but only the configuration class (name), no?

Reply:
Any suggestions?

Reply:
Maybe "Using fork starter with configuration implementation " to be in line with https://github.com/apache/maven-surefire/blob/0998f10bb486f79f40d2d4262798b1f53a5ff286/maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java#LL2228C32-L2228C32?